src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -270,16 +283,17 @@ private static HttpRequestRetryHandler getHttpRequestRetryHandler() {
         LOGGER.info("Max retry exceeded for requestURI:{}", requestURI);
         return false;
       }
-      if (exception instanceof NoHttpResponseException) {
+      if (exception instanceof NoHttpResponseException
+          || exception instanceof javax.net.ssl.SSLException) {
         LOGGER.info(
-            "Retrying request which caused No HttpResponse Exception with "
-                + "URI:{}, retryCount:{} and maxRetryCount:{}",
+            "Retrying request which caused {} with " + "URI:{}, retryCount:{} and maxRetryCount:{}",
+            exception.getClass().getName(),
```

Comment:
Codecov says it is not covered by tests.

Reply:
Mockito tests of `HttpRequestRetryHandler` added.